poll.js

```diff
@@ -0,0 +1,67 @@
+// Description:
+//   Poll_bot
+//
+// Dependencies:
+//   None
+//
+// Configuration:
+//   None
+//
+// Commands:
+//   !poll, "question?", "option 1", "option 2" 
+//
+
+module.exports = function (robot) {
```

Comment:
It's strange. I've known from docs when you do function export, you must specify function name.

![selection_019](https://user-images.githubusercontent.com/34273859/43457350-3e717bf4-94cf-11e8-9eb5-514d9935e440.png)

Reply:
It's ok, look here, for example: https://github.com/tolstoyevsky/hubot-happy-birthder/blob/2e76faa96e276de8a586f3d4695129ad6ab7fe9f/index.js#L9